src/offline/utils.ts

```diff
@@ -0,0 +1,67 @@
+import { deriveKeypair, deriveAddress, deriveXAddress } from './derive'
+import computeLedgerHeaderHash from './ledgerhash'
+import signPaymentChannelClaim from './sign-payment-channel-claim'
+import verifyPaymentChannelClaim from './verify-payment-channel-claim'
+import { dropsToXrp,
+    xrpToDrops,
+    toRippledAmount,
+    convertKeysFromSnakeCaseToCamelCase,
+    removeUndefined,
+    rippleTimeToISO8601,
+    iso8601ToRippleTime,
+    isValidSecret } from '../common/utils'
+import {
+    computeBinaryTransactionHash,
+    computeTransactionHash,
+    computeBinaryTransactionSigningHash,
+    computeAccountLedgerObjectID,
+    computeSignerListLedgerObjectID,
+    computeOrderID,
+    computeTrustlineHash,
+    computeTransactionTreeHash,
+    computeStateTreeHash,
+    computeLedgerHash,
+    computeEscrowHash,
+    computePaymentChannelHash,
+} from '../common/hashes'
+import { generateAddressAPI, GenerateAddressOptions, GeneratedAddress } from '../offline/generate-address'
+
+// @deprecated Use X-addresses instead
+const generateAddress = (options: GenerateAddressOptions = {}): GeneratedAddress => {
+    return generateAddressAPI({...options, includeClassicAddress: true})
+}
+
+const generateXAddress = (options: GenerateAddressOptions = {}): GeneratedAddress => {
+    return generateAddressAPI(options)
+}
```

Comment:
ditto

Reply:
actually, even simpler:

```javascript
const generateXAddress = generateAddressAPI
```

Reply:
Ah right, since they have the same parameters. I'll make this change, thanks!